src/watchers/member_status.js

```diff
@@ -116,7 +116,10 @@ class MemberStatusHandler {
         else if (doc.table === 'candidates'){
             account = doc.data.candidate_name;
         }
-        else if (doc.table === 'custodians'){
+        else if (
+            (doc.code === 'dao.worlds' && doc.table === 'custodians1') ||
```

Comment:
as above. Keep it simple. Sorry maybe my direction was more confusing that it needed to be.